### ISSUE TITLE
SMB version 1.0 fails to be autodetected by smbtree

### ISSUE THREAD

**Issue by zebradots** [redorescue/redorescue#15]
When detecting network shares, those running the outdated SMB v1.0 protocol are not detected automatically by the `smbtree` command. There may be no way to easily overcome this, but this issue has been opened in case `smbtree` has a documented (or hidden) switch to locate older shared resources.

If they cannot be detected, they will simply need to be entered manually.

**Comment by zebradots** [comment]
Evidently there is a reason `smbtree` doesn't find these network shares. After researching the status of SMB 1.0, we have decided not to provide support going forward. In the end this was not a decision based on complexity or UX issues, but because it is outdated and has been deprecated.

SMB 1.0 was developed in the 1990s and has many vulnerabilities. The infamous WannaCry ransomware exploited flaws in the SMB 1.0 protocol due to its insecurity, but before this made headlines, Microsoft made the decision to retire SMB 1.0 public in 2014. As of 2017, Windows 10 support for SMB 1.0 was removed and is now no longer available on new installs.

> "[Right now, SMBv]2.02 is the version that ships with Windows Server 2008, which is the oldest supported OS in the Windows/Windows Server stable currently. That makes it the minimum recommended," the expert added. "We prefer that everyone run SMB 3.1.1 as a minimum, as it is the most secure and has the most functionality," Pyle says. "SMB1 is deprecated, meaning it is effectively abandoned except for security updates."

As if this weren't enough, SMB 1.0 is slower and provides less stable connections, which are both likely to cause problems when attempting a backup or restore process with Redo. **In light of the foregoing, we have decided to follow Microsoft's lead and bid farewell to SMB 1.0: Redo will not provide compatibility with this deprecated protocol.**

Source: https://www.bleepingcomputer.com/news/microsoft/microsoft-to-disable-smbv1-in-windows-starting-this-fall/
